recipes/core/busybox.yaml

```diff
@@ -77,5 +77,9 @@ buildScript: |
         kconfigIsSet CONFIG_SHELL_HUSH build/.config && printf '/bin/hush\n/usr/bin/hush\n' ||:
     ) > install/.bob/busybox.shell-table
 
+    # install udhcpc script if enabled
+    kconfigIsSet CONFIG_UDHCPC build/.config && \
+        install -D -m 0755 $<@busybox/udhcpc.script@> install/usr/share/udhcpc/default.script ||:
```

Comment:
I think we should avoid this pattern in the future. If the `install` command fails for whatever reason, we won't notice. So the proper way is

```sh
if kconfigIsSet CONFIG_UDHCPC build/.config ; then
    install -D -m 0755 $<@busybox/udhcpc.script@> install/usr/share/udhcpc/default.script
fi
```

I know I didn't object before so I'm fine with it in this PR. Just that this is not used anywhere else in the future.